### PR TITLE
Handle 'cities' and 'test_counter' for ArtemisNG

### PR DESCRIPTION
**Pull request by mbounabi**

### CODE REVIEW

File: artemis/base_pytest.py [hunk 1]
```diff
@@ -254,8 +253,34 @@ def get_file_name(self):
...
+        if len(self.__class__.data_sets) == 1:
+            full_url = "coverage/{region}/{url}".format(region=self.__class__.data_sets[0].name, url=url)
+
+        return self._api_call(full_url, response_checker)
+
+    def _api_call(self, url, response_checker):
+        """
+        call the api and check against previous results
+
+        the query is writen in a file
```
Comment (pbougue) [inline artemis/base_pytest.py]: minor :gb:: writ`t`en

Comment (mbounabi) [reply artemis/base_pytest.py]: Will be corrected in the next PR.

File: artemis/conftest.py [hunk 2]
```diff
@@ -29,7 +30,13 @@ def load_cities(request):
     """
     log = logging.getLogger(__name__)
     if request.config.getvalue("skip_cities") or request.config.getvalue("check_ref"):
-        log.info("skiping cities loading")
+        log.info("skipping cities loading")
+        return
+
+    if config['USE_DOCKER']:
+        url = "http://localhost:9898/v0/cities/"
```
Comment (xlqian) [inline artemis/conftest.py]: it'd be better to put this in a config

Comment (mbounabi) [reply artemis/conftest.py]: done in a PR

**Pull request merged**